src/helpers/custom.js

```diff
@@ -139,13 +140,38 @@ const customBoxShadowObjectFormatter = (dictionary, theme, isJS) => {
   );
 };
 
+const customOpacityObjectFormatter = (dictionary, theme, isJS) => {
+  // no opacity tokens for core, returns empty object otherwise
+  if (theme?.destination.includes("core")) return "";
+
+  const opacityTokens = dictionary.allTokens.filter(
+    (token) => token.attributes.category === "opacity"
+  );
+  const valueOrType = (token, isJS) => {
+    return isJS
+      ? `"${token.value}"`
+      : `string`;
+  };
+
+  return (
+    declaration(isJS) +
+    `opacity: {` +
+    opacityTokens.map((token) => {
+      return `${token.name} : ` + valueOrType(token, isJS);
+    }) +
+    `}` +
+    commaOrColon(isJS)
+  );
+};
+
```

Comment:
Y'all are transforming aces now ♠️ 😎

Reply:
love to see it. we could def clean up this file but thats a battle for another day 🥲

Reply:
Agree! Just copy and pasting at this point 😅